lms/static/sass/elements/_controls.scss

```diff
@@ -403,3 +403,22 @@
       margin-bottom: none;
     }
   }
+
+  .btn-link {
```

Comment:
@marcotuts FYI, This Sass is for Leave Team button. The goal was to show the button as link.